XamarinCommunityToolkit/Behaviors/MaskedBehavior.cs

```diff
@@ -0,0 +1,87 @@
+﻿using System;
+using System.Collections.Generic;
+using Xamarin.Forms;
+
+namespace Microsoft.Toolkit.Xamarin.Forms.Behaviors
+{
+    public class MaskedBehavior : Behavior<Entry>
+    {
+        string mask = "";
+        IDictionary<int, char> positions;
+
+        public string Mask
+        {
+            get => mask;
+            set
+            {
+                mask = value;
+                SetPositions();
+            }
+        }
+
+        protected override void OnAttachedTo(Entry entry)
+        {
+            entry.TextChanged += OnEntryTextChanged;
+            base.OnAttachedTo(entry);
+        }
+
+        protected override void OnDetachingFrom(Entry entry)
+        {
+            entry.TextChanged -= OnEntryTextChanged;
+            base.OnDetachingFrom(entry);
+        }
+
+        void SetPositions()
+        {
+            if (string.IsNullOrEmpty(Mask))
+            {
+                positions = null;
+                return;
+            }
+
+            var list = new Dictionary<int, char>();
+            for (var i = 0; i < Mask.Length; i++)
+            {
+                if (Mask[i] != 'X')
```

Comment:
Mask character should be customizable, so please make a bindable property for it as well

Reply:
ye no problem